src/agent/onefuzz-supervisor/src/worker/tests.rs

```diff
@@ -67,6 +67,22 @@ pub struct ChildDouble {
     killed: bool,
 }
 
+impl Default for ChildDouble {
+    fn default() -> Self {
```

Comment:
Idea: instead of making a different impl for `Default`, which was already being derived, just add another ctor to the impl block for `ChildDouble`, like:
```rust
impl ChildDouble {
    fn exited_success() -> Self { 
        // Use the derived `Default` impl here, then update it to give it a non-`None` exit status
    }
}
```

Reply:
removed, it was not needed for my changes